README.md

```diff
@@ -1,153 +1,96 @@
 
-# jupyter_ascending
+# Jupyter Ascending
 
 Sync Jupyter Notebooks from any editor
 
 ![Jupyter Ascending](./media/simple_jupyter_ascending.gif)
```

Comment:
This gif could be much better. It's really hard to see what is going on. Should be larger text, very little text, very short, and with very descriptive code (ex: add a line like "print('I'm writing this line in vim and it shows up in jupyter!', then sync, then add another: "print('and you can run cells too')", then sync and run, then wait at least 5 seconds so they can figure out what just happened)"